_data/compatibility/trezor.yaml

```diff
@@ -0,0 +1,68 @@
+---
+name: Trezor
+internal_url: /en/compatibility/trezor
+logo: /img/compatibility/trezor/trezor.png
+rbf:
+  tested:
+    date: "2018-11-08"
+    platforms:
+      - web
+    version: "n/a"
+  features:
+    receive:
+      notification: "na"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "true"
+      shows_original_version: "false"
+    send:
+      signals_bip125: "false"
+      list: "false"
```

Comment:
how are these things testable (sent transaction labeled as RBF in summary and details screen), when the service doesn't support sending with RBF?

Reply:
You’re right. "Doesn’t flag RBF transactions (incoming)" doesn’t mean "Doesn’t flag RBF transactions outgoing". 

I will make sure this is consistent across other data files as well.